alethzero/plugins/whisper/Whisper.cpp

```diff
@@ -53,6 +55,17 @@ static Public stringToPublic(QString const& _a)
 		return Public();
 }
 
+static shh::Topics stringToTopics(QString _s)
+{
+	shh::BuildTopic ret;
+	QStringList tx = _s.split("|", QString::SkipEmptyParts);
```

Comment:
I haven't run this code to see the UI. Is this input guaranteed to be hex delimited by |, or is it any set of bytes?